storey/targets.py

```diff
@@ -509,6 +520,7 @@ def __init__(
         columns: Union[str, Union[List[str], List[Tuple[str, str]]], None] = None,
         partition_cols: Union[str, Union[List[str], List[Tuple[str, int]]], None] = None,
         time_field: Union[None, str, int] = None,
+        time_format: Optional[str] = None,
```

Comment:
add to ParquetTarget documentation

Reply:
Man, I knew I was going to forget this...